README.md

```diff
@@ -1,2 +1,58 @@
-# indevelopme-coaching-publications
-These are the publications that players on our team created for publications.
+# indevelopme-coaching-publications  
+
+This repository holds a free collection of IT knowledge that has been created and reviewed for publication into Medium / Linkedin, and other platforms. Participants in the JustAskDavidB Coaching Program (aka GSD-IT) maintain this repository to showcase their IT compentency with hopes of improving IT literacy in non-IT communities. Highly consistent contributors have a chance to be offered paid opportunity which includes free self-training products such as Linkedin Learning, Coursera, etc. and opportunities to work on paid projects for inDevelopme. 
+
+## Purpose  
+
+The JustAskDavidB Coaching Program intends to help fives groups of people develop understanding of IT concepts at an accelerated rate:
+- individuals zero prior IT experience  
+- individuals pursuing an IT degree  
+- individuals seek a career shift into IT  
+- mid and senior level IT professions trying to enter management  
+
+## Description  
+
+Participants will work under the guidance of a coach to complete open-source initiatives for inDevelopme's, with tasks being specific to the individual's selected coaching track (e.g. Programming, DevOps, Content Strategy). When necessary, participants shall be granted access to software licenses, credentails, and information to complete tasks.
+
+## Tracks 
+
+Participants can choose to receive guidance from 1 of 3 GSD-IT categories:
+- Programming
+- DevOps
+- Content Strategy
+
+### Programming
+
+This GSD-IT track focuses heavily on programming concepts and best practicing. GIT, Composer, Jetbrain, Laravel (PHP), and Flask (Python) represent some of the products and technologies used within this track. 
+
+***Programming: General Tasks***  
+
+Participants will work through a series of tasks meticulously designed to foster growth in programming languages (e.g. Python, Java, C++, etc). They will tackle debugging scenarios, conditional statements, iterative programming,practice user input and output handling, and build basic user interfaces. Other tasks will include self-exploration in data structures, algorithms, version control, etc to participant on small projects designed to allow application of newly acquired knowledge and skills.  
+
+***Programming: Benefits to Participant***  
+
+Consistent engagement in programming will help partcipants build a solid foundation in programming, enabling them to take on more complex challenges. Participants will also develop a portfolio of skills and work experience for the world to see!
+
+### DevOps  
+
+This GSD-IT track focuses heavily on devops concepts and best practices using AWS. Docker, Jenkins, Terraform, and Amazon Web Service (AWS) are just some of the products and technologies used within this track.
```

Comment:
- should be "DevOps"
- spell out AWS on first usage with AWS in parentheses, and then use AWS on second reference